### DISCUSSION TITLE
VREPEAT_NODE耗时超高

### DISCUSSION THREAD

**Discussion by Larborator** [apache/doris#21785] [A - General / Q&A] [answered]
![1689230292295](https://github.com/apache/doris/assets/19779893/cd7937b4-fc8a-47ea-aef8-aee0705d69f6)

有大佬知道这个node是做什么的吗？测试过这个节点的磁盘读写速度是正常的（200MB/s），但是VREPEAT_NOD处理180M数据用了64秒

**Comment by yagagagaga** [comment, marked as answer]
[Here](https://doris.apache.org/zh-CN/community/design/grouping_sets_design/) is the design doc you can refer.
